portfolio/src/main/java/com/google/sps/comment/data/Comment.java

```diff
@@ -8,24 +8,28 @@
 public final class Comment implements EntityConvertible {
     private static final String KIND = "Comment";
```

Comment:
Both CommentDataStore and Comment has this constant and they need to be the same. You should define in one only as it's not clear who has the responsibility to update it if it's changed. Maybe you can define in the comment data store and pass in via the "toEntity" below.

Reply:
Fixed